### ISSUE TITLE
Multiple Video-Text pair Support

### ISSUE THREAD

**Issue by mustafaadogan** [OpenGVLab/Ask-Anything#78]
Hello! 

First of all, I'd like to congratulate you on your great work. I have a question: I'm looking to evaluate the model's performance in a different way by using in-context examples. Specifically, I'm interested in feeding the model multiple in-context video-text examples. Is it possible to do so?

**Comment by Andy1621** [comment]
Yes.  I have tried to conduct in-context inferring but found it hard to follow instructions.  I doubt that it may be because of the lack of similar tuning data.  If you are interested, maybe you can follow [MIMIC](https://github.com/HaozheZhao/MIC) to design the data.

**Comment by yinanhe** [comment]
Due to the lack of updates for a long time, your issue has been temporarily closed. If you still have any problems, please feel free to reopen this issue.
